ui/src/app/store/machine/types.ts

```diff
@@ -212,7 +212,7 @@ export type MachineMetadata = {
   system_version?: string;
 };
 
-export type MachineNumaNode = {
+export type MachineNumaNode = Model & {
```

Comment:
Not sure if this changed in the api or if I missed it the first time :man_shrugging: